### PR TITLE
refactor(map_height_fitter): rework parameters

### PR DESCRIPTION
**Pull request by TaikiYamada4**
## Description

1. Extract parameters from `map_height_fitter.launch.xml` and create `map_height_fitter.param.yaml` file for map_height_fitter.
2. Create `map_height_fitter.schema.json` for the `map_height_fitter.param.yaml` file. 
3. Also rewrite README.md to read `map_height_fitter.schema.json`.

### 1. Passing ros parameters
Confirmed that `ros2 param dump /map/map_height_fitter` output the following
```
/map/map_height_fitter:
  ros__parameters:
    map_loader_name: /map/pointcloud_map_loader
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
```

### JSON validation
Tested that the json schema is valid in local environment by
```
find -wholename '*/schema/*.schema.json' -printf '%p: ' -execdir bash -c 'check-jsonschema --schemafile "$1" ../config/"${1:2:-12}"*.param.yaml' bash '{}' +
```

### Write a table of ros parameters via `map_projection_loader.schema.json`
I did `mkdocs serve` and confirmed that the table of ros parameters is constructed in the documents.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: map/map_height_fitter/config/map_height_fitter.param.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/**:
+  ros__parameters:
+    map_loader_name: /map/pointcloud_map_loader
```
Comment (Motsu-san) [inline map/map_height_fitter/config/map_height_fitter.param.yaml]: @TaikiYamada4 Keeping it consistent to other param.yaml files, could you make the value double-quoted?

**Review by Motsu-san — CHANGES_REQUESTED** [review]
@TaikiYamada4 There's a small change request. Could you?

File: map/map_height_fitter/config/map_height_fitter.param.yaml (hunk 1 above)
Comment (TaikiYamada4) [reply map/map_height_fitter/config/map_height_fitter.param.yaml]: Indeed. I fixed it now. Thank you for the comment!

**Review by Motsu-san — APPROVED** [review]
[LGTMeow](https://lgtmeow.com)

**Pull request merged**